handler.py

```diff
@@ -0,0 +1,74 @@
+import boto3
+import cv2
+import os
+import numpy as np
+from io import BytesIO
+
+s3 = boto3.resource('s3')
+
+
+def opencv(event, context):
+    bucketName = event['Records'][0]['s3']['bucket']['name']
+    objectKey = event['Records'][0]['s3']['object']['key']
+    img_obj = s3.Object(
+        bucket_name=bucketName,
+        key=objectKey,
+    )
+    obj_body = img_obj.get()['Body'].read()
+
+    try:
+     
+        img = cv2.imdecode(np.asarray(bytearray(obj_body)), cv2.IMREAD_COLOR)
+
+        smoothed_image = cv2.GaussianBlur(img, (9, 9), 10)
+        unsharped_image = cv2.addWeighted(img, 1.5, smoothed_image, -0.5, 0)
+        
+        #improve image contrast
+        clahe = cv2.createCLAHE(clipLimit=4.0)
+
+        H, S, V = cv2.split(cv2.cvtColor(unsharped_image, cv2.COLOR_RGB2HSV))
+        eq_V = clahe.apply(V)
+        eq_image = cv2.cvtColor(cv2.merge([H, S, eq_V]), cv2.COLOR_HSV2RGB)
```

Comment:
Can we add a comment or two here to explain what's being done?

Reply:
Done